src/modules/tntt/ui/views/TnttDetail.tsx

```
@@ -5,104 +5,161 @@ import { useParams } from "next/navigation";
+              src={tntt.thumbnail}
+              alt={tntt.title}
+              width={800}
+              height={400}
+              className="rounded-lg mb-6 w-full object-cover max-h-72 sm:max-h-96"
+            />
           )}
+          <div
+            className="text-gray-900 font-medium leading-relaxed max-w-none text-wrap text-justify text-base sm:text-lg"
+            dangerouslySetInnerHTML={{
+              __html: htmlDescription || "Không có nội dung",
+            }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)